docs/src/solvers/pinn.md

```diff
@@ -26,7 +26,7 @@ discretization = PhysicsInformedNN(chain,
                                    nothing; #init_params
                                    phi = nothing,
                                    derivative = nothing,
-                                   strategy = GridTraining())
+                                   strategy = nothing)
```

Comment:
hmm, if strategy is required, shouldn't it be a required argument?

Reply:
yeap, you right